### ISSUE TITLE
Failed to build rocksdb-node on ubuntu

### ISSUE THREAD

**Issue by zhaoyi0113** [dberesford/rocksdb-node#6]
I failed to build it on ubuntu linux with below errors. The version is `"rocksdb-node": "^1.7.1"`. 

```
make: Entering directory '/workspace/TheWebServer.bak/node_modules/rocksdb-node/build'
  CXX(target) Release/obj.target/rocksdb/rocksdb.o
In file included from ../rocksdb.cc:3:0:
../DBNode.h:5:24: fatal error: rocksdb/db.h: No such file or directory
compilation terminated.
rocksdb.target.mk:108: recipe for target 'Release/obj.target/rocksdb/rocksdb.o' failed
make: *** [Release/obj.target/rocksdb/rocksdb.o] Error 1
make: Leaving directory '/workspace/TheWebServer.bak/node_modules/rocksdb-node/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:194:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
gyp ERR! System Linux 4.9.49-moby
gyp ERR! command "/usr/bin/nodejs" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /workspace/TheWebServer.bak/node_modules/rocksdb-node
gyp ERR! node -v v7.10.1
gyp ERR! node-gyp -v v3.5.0
gyp ERR! not ok 
```

**Comment by zhaoyi0113** [comment]
I found out that the issue relates to rocksdb header file. After clone rocksdb and build it on ubuntu. The `rocksdb/db.h` error is gone but a new error comes up like below. I have installed this file in `/usr/lib/librocksdb.a ` but it still can't find it. 

```
/usr/local/include/rocksdb/sst_file_writer.h:93:34: note: declared here
   ROCKSDB_DEPRECATED_FUNC Status Add(const Slice& user_key, const Slice& value);
                                  ^
  CXX(target) Release/obj.target/rocksdb/DBNode.o
../DBNode.cc: In static member function 'static Nan::NAN_METHOD_RETURN_TYPE DBNode::CreateColumnFamily(Nan::NAN_METHOD_ARGS_TYPE)':
../DBNode.cc:557:7: warning: variable 'optsIndex' set but not used [-Wunused-but-set-variable]
   int optsIndex = -1;
       ^
  CXX(target) Release/obj.target/rocksdb/DBWithTTL.o
  CXX(target) Release/obj.target/rocksdb/CompactRangeWorker.o
  CXX(target) Release/obj.target/rocksdb/MultiGetWorker.o
  SOLINK_MODULE(target) Release/obj.target/rocksdb.node
g++: error: librocksdb.a: No such file or directory
rocksdb.target.mk:137: recipe for target 'Release/obj.target/rocksdb.node' failed
```

**Comment by dberesford** [comment]
Thanks for fixing this @zhaoyi0113, fix published in rocksdb-node@1.7.2.
